apache2/apache2_config.c

```diff
@@ -125,6 +125,11 @@ void *create_directory_config(apr_pool_t *mp, char *path)
     dcfg->stream_inbody_inspection = NOT_SET;
     dcfg->stream_outbody_inspection = NOT_SET;
 
+    /* WAF policy identification information */
+    dcfg->waf_policy_id = NOT_SET_P;
+    dcfg->waf_policy_scope = NOT_SET_P;
```

Comment:
I have added a comment explaining that 'site' field is deprecated. Let's not remove it for the time being to avoid unexpected regressions at customers' side where they may have logs parsing rely on it.